core/src/main/scala/org/ensime/core/RichPresentationCompiler.scala

```diff
@@ -145,21 +145,6 @@ trait RichCompilerControl extends CompilerControl with RefactoringControl with C
   def askReloadAndTypeFiles(files: Iterable[SourceFile]) =
     askOption(reloadAndTypeFiles(files))
 
-  def askUsesOfSymAtPos(pos: Position)(implicit ec: ExecutionContext): Future[List[RangePosition]] = {
```

Comment:
I'm wondering if perhaps we should still augment the indexer-only results with the hits from the currently open source files. Part of me prefers the new cleaner "indexer only" impl, but the other half knows that somebody is going to expect it to show hits in their current file.